contrib/kafka/cmd/dispatcher/main.go

```diff
@@ -55,16 +59,30 @@ func main() {
 		logger.Fatal("Unable to add kafkaDispatcher", zap.Error(err))
 	}
 
-	if err := v1alpha1.AddToScheme(mgr.GetScheme()); err != nil {
+	if err = v1alpha1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
why not make err scoped to "if".

Reply:
I have a general preference to not alias variables, but I know this is controversial, so will change it if you want. In particular aliasing `err` (when handled in the normal go way) is not likely to cause any problems, so I'm willing to change it.

Reply:
Done.